components/colorpalette/overview.md

```diff
@@ -10,102 +10,85 @@ position: 0
 
-## Basics
+## Creating Blazor RadioGroup
 
-To use a Telerik Color Palette for Blazor:
+1. Add the `<TelerikColorPalette>` tag to a Razor file.
+1. Set the `Value` parameter to a `string` object. It supports one-way and two-way binding.
 
-1. Add the `<TelerikColorPalette>` tag.
-1. Bind its `Value` to the `string` you want to get out of it.
-1. Optionally, choose a list of `Colors` to show the user (one of the [presets we provide]({%slug colorpalette-presets%}), or a set of [custom colors]({%slug colorpalette-custom-colors%})).
-    * If you do not provide a value for the `Colors`, it will default to the `Office` [preset]({%slug colorpalette-presets%}).
-
->caption Basic color palette with two-way value binding and a default predefined palette
+>caption Basic color palette with two-way value binding and a default predefined palette.
 
 ````CSHTML
-<span style="color: @MyColor">@MyColor</span>
-<br />
+<TelerikColorPalette @bind-Value="@MyColor" />
 
-<TelerikColorPalette @bind-Value="@MyColor">
-</TelerikColorPalette>
+<p>Selected color: <span style="color: @MyColor">@MyColor</span></p>
 
 @code {
-    public string MyColor { get; set; }
+    string MyColor { get; set; }
 }
 ````
 
->caption The result from the code snippet above after selecting a color
+## Predefined Colors
 
-![Color Palette first look](images/color-palette-first-look.png)
+The ColorPalette component comes with a set of predefined colors that are shown to the users. [Read more about the available ColorPallete predefined colors]({%slug colorpalette-presets%}).
 
-## Appearance
+## Custom Colors
 
-You can control the appearane of the component not only through the lists of `Colors` you provide to it, but also its size through the `Columns`, `TileWidth` and `TileHeight` parameters.
+The ColorPallete can work with your own set of colors. [Read more about the Blazor ColorPallete custom colors setup]({%slug colorpalette-custom-colors%}).
 
->caption Make a large color palette with few columns
+## Events
 
-````CSHTML
-@SelectedColor
-<TelerikColorPalette Colors="@ColorPalettePresets.Basic" @bind-Value="@SelectedColor"
-                     Columns="5" TileHeight="3em" TileWidth="3em">
-</TelerikColorPalette>
-@code{
-    string SelectedColor { get; set; }
-}
-````
+The Blazor ColorPalette fires value change and blur events that you can handle and further customize its behavior. [Read more about the Blazor Dialog events]({%slug colorpalette-events%}).
 
->caption Theresult from the code snippet above
+## ColorPalette Parameters
 
-![color palette appearance and size customization](images/large-size-few-columns.png)
+The Blazor ColorPalette provides various parameters to configure the component:
 
+<style>
+    article style + table {
+        table-layout: auto;
+        word-break: normal;
+    }
+</style>
+| Parameter | Type and Default Value | Description |
+| ----------- | ----------- | ----------- |
```

Comment:
Either format the table via ASCII art, or use minimal design. No need for something in between.
```suggestion
| --- | --- | --- |
```